src/submission/validation-clinical/utils.ts

```diff
@@ -59,8 +59,8 @@ export const buildSubmissionUpdate = (
     index,
     info: {
       donorSubmitterId: newRecord[DonorFieldsEnum.submitter_donor_id],
-      newValue: `${newRecord[fieldName]}`, // we convert the value to string since lectern may converted it to non string (integer, boolean)
-      oldValue: `${oldValue}`,
```

Comment:
if the value is null this returned "null", not sure if thats what we wanted?

Reply:
I don't quite remember the order of how things changed, but this may have happened before the changes in lectern where I stopped it from populating default values to unsubmitted fields